CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - On Wayland, DPI calculation is handled by smithay-client-toolkit.
 - On X11, `WindowBuilder::with_min_dimensions` and `WindowBuilder::with_max_dimensions` now correctly account for DPI.
 - Added support for generating dummy `DeviceId`s and `WindowId`s to better support unit testing.
+- **Breaking:** Removed minimum supported Rust version guarantee.
```

Comment:
I'm not sure how much of a breaking change this is. I mean, compatibility is already broken in 0.18.0...

On that same note, should we remove the entry from 0.18.0 about the 1.28.0 change? That was something I added *after* release, so it's always felt a bit weasely, and it's not even true anymore.

Reply:
If we remove the `1.28.0` line from the changelog, I feel like we should remove all MSRV lines from the changelog. I'd be fine with doing that, but I don't feel too strongly either way.

Reply:
Do we have any other MSRV lines in the changelog, though? The only other mention of MSRV changes I see is under 0.10.0, and that feels a bit different to me.

Reply:
Eh, fair enough. Just to be clear about where we stand: this shouldn't be marked as a breaking change but still be in the changelog, and the `1.28` change should be removed?

Reply:
That's my position, yes.

Reply:
SGTM. Those changes have been made.